commit 4487d1032ebbeaed0b7670010ded91ed9ec5702f (refs/changes/71/671/3)
Author: David Pacheco <dap@joyent.com>
Date:   2016-10-13T21:31:45+00:00 (3 years ago)
    
    MORAY-367 moray dependents should not depend on master
    Reviewed by: Joshua M. Clulow <jmc@joyent.com>
    Approved by: Joshua M. Clulow <jmc@joyent.com>

diff --git a/dev/package.json b/dev/package.json
index dd50c2f..83f1d7c 100644
--- a/dev/package.json
+++ b/dev/package.json
@@ -32,7 +32,7 @@
 		"manta-compute-bin": "git+https://github.com/joyent/manta-compute-bin.git#master",
 		"memorystream": "0.2.0",
 		"mkdirp": "0.3.1",
-		"moray": "git+https://github.com/joyent/node-moray.git#master",
+		"moray": "git+https://github.com/joyent/node-moray.git#fd5781bc25a9bfe2ba82167664639753fb9f0ca5",
 		"native-dns": "0.2.1",
 		"node-uuid": "1.4.0",
 		"panic": "0.2.1",
diff --git a/package.json b/package.json
index f5a20a2..84bdc51 100644
--- a/package.json
+++ b/package.json
@@ -22,7 +22,7 @@
 		"jsprim": "0.5.0",
 		"lstream": "0.0.3",
 		"mahi": "git+https://github.com/joyent/node-mahi.git#master",
-		"moray": "git+https://github.com/joyent/node-moray.git#master",
+		"moray": "git+https://github.com/joyent/node-moray.git#fd5781bc25a9bfe2ba82167664639753fb9f0ca5",
 		"node-uuid": "1.4.0",
 		"posix-getopt": "1.0.0",
 		"retry": "0.6.0",
